e2e/support/helpers/e2e-relative-date-picker-helpers.js

```diff
@@ -47,8 +47,9 @@ function addStartingFrom({ value, unit }) {
 }
 
 function toggleCurrentInterval() {
-  popover().findByLabelText("Options").click();
-  popover().last().findByTestId("include-current-interval-option").click();
+  popover()
+    .findByTestId("include-current-interval-option")
+    .click({ force: true });
```

Comment:
As an alternative, I can use [the other pattern](https://github.com/metabase/metabase/pull/44179/files#diff-1195543a3323e8a1a06046b084cfc8a16544ae157a7083691a9f90d7b27a9f93R525) and find the actual text in the UI. Either way is fine with me.